recipes/simbody/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+source activate "${CONDA_DEFAULT_ENV}"
+
+mkdir build
+cd build
+cmake .. \
+	-LAH \
+	-DCMAKE_INSTALL_PREFIX="$PREFIX" \
+	-DBUILD_USING_OTHER_LAPACK="$PREFIX/lib/libopenblas.so" \
+	-DBUILD_VISUALIZER=off
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
+	make -j$(nproc)
+	ctest -j$(nproc)
+	make -j$(nproc) install
```

Comment:
That's a fair question. This is one of the reasons I want us to find a way to override `CPU_COUNT`. ( https://github.com/conda/conda-build/issues/1117 ).

For now `MAKEFLAGS` is defined for auto passthrough. So, you could set `MAKEFLAGS="-j$(proc)"` on your own infrastructure and it will automatically be included in your scripts just as it is at conda-forge.